EKrut-Client/src/ekrut/client/gui/OrderReportViewController.java

```diff
@@ -47,24 +60,50 @@ public class OrderReportViewController implements Initializable{
     @FXML
     private Label totalOrdersLbl;
 
-    
-    private EKrutClient client;
+    private EKrutClient client = EKrutClientUI.getEkrutClient();
     ClientReportManager  clientReportManager = new ClientReportManager(client);
```

Comment:
The client already has an instance of the report manager. You should get that instance by using `client.getClientReportManager`.